### PR TITLE
Small update

### PR DESCRIPTION
**Pull request by Eqis-Edu**
+ Another text fix #1236 to datagen
+ Add hidden ritual advancements #1238 
+ More text consistencies #1240 (wild/possessed evoker, also to breezes)
+ Add a page in Dimensional Mineshaft entry about config for miner go to output before break (not exactly #1241 wants, but it's what I offer)
+ Add a page in Divination Rod entry about config to locate any ore
+ Add a note in Ronaza's entry about the iron rod not rendering
+ Removed outdated quotes about decay
+ Item 10 from #1155 (purified ink texture)
+ Familiar ring texture change familiar_ring
+ Bone meal crushing recipe
+ Slight change in width of spirits hitbox to fit in one block
+ Rozana's Contact blocks can be placed in ritual satchel
+ Removed remaing non-pentacles entries advancement locks
+ Add missing food item tag
+ Add Marid Essence to miner and stabilizer recipe
+ Add Afrit Essence to stabilizer recipe
+ Change the Afrit Crusher recipe
      + Iesnium, emeral, lapis, amethyst, obsidian (all dust)
+ EMI integration news:
      + Crushing show min and max tier in tooltip, the min is visual too
      + Show the repair ritual when looking for the craft of a valid item (because repair in crafting/anvil/grindstone appers)
      + Show extra drops from possessed mobs (hardcoded for now, trying todo better) 
+ Changed chalks advancements structure to work better in ATM and others modpacks
      
 Pictures:
 

(ATM testing, need changing the slot some times but work)

### CODE REVIEW

**Comment by klikli-dev** [comment]
@Eqis-Edu can you ping me when it is ready to merge? I don't want to pull the trigger while you are still in the middle of things!

**Comment by Eqis-Edu** [comment]
@klikli-dev Now is ready!

**Comment by MuteTiefling** [comment]
> Add a page in Divination Rod entry about config to locate any ore

Why not just change this (and the nearly invisible glass plane) to something else?

This message will go unnoticed and players will still be left confused.

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 1]
```diff
@@ -1487,7 +1491,7 @@
   "item.occultism.ritual_dummy.craft_storage_remote": "Ritual: Craft Storage Accessor",
   "item.occultism.ritual_dummy.craft_storage_remote.tooltip": "The Storage Accessor can be linked to a Storage Actuator to remotely access items.",
   "item.occultism.ritual_dummy.craft_witherite_dust": "Ritual: Craft Witherite Dust",
-  "item.occultism.ritual_dummy.craft_witherite_dust.tooltip": "A afrit will infuse wither essence in the netherite dust.",
+  "item.occultism.ritual_dummy.craft_witherite_dust.tooltip": "An afrit will infuse netherite dust with wither essence.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:1494]: Afrit should be capitalized to fit with the rest of the books written style.

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 2]
```diff
@@ -2174,9 +2178,9 @@
   "ritual.occultism.possess_witch.interrupted": "Summoning of possessed witch interrupted.",
   "ritual.occultism.possess_witch.started": "Started summoning possessed witch.",
   "ritual.occultism.possess_zombie_piglin.conditions": "Not all requirements for this ritual are met.",
-  "ritual.occultism.possess_zombie_piglin.finished": "Summoned possessed zombie piglin successfully.",
-  "ritual.occultism.possess_zombie_piglin.interrupted": "Summoning of possessed zombie piglin interrupted.",
-  "ritual.occultism.possess_zombie_piglin.started": "Started summoning possessed zombie piglin.",
+  "ritual.occultism.possess_zombie_piglin.finished": "Summoned possessed zombified piglin successfully.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2181]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 3]
```diff
@@ -2174,9 +2178,9 @@
...
+  "ritual.occultism.possess_zombie_piglin.interrupted": "Summoning of possessed zombified piglin interrupted.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2182]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 4]
```diff
@@ -2174,9 +2178,9 @@
...
+  "ritual.occultism.possess_zombie_piglin.started": "Started summoning possessed zombified piglin.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2183]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 5]
```diff
@@ -2294,9 +2298,9 @@
   "ritual.occultism.summon_foliot_transport_items.interrupted": "Summoning of foliot transporter interrupted.",
   "ritual.occultism.summon_foliot_transport_items.started": "Started summoning foliot transporter.",
   "ritual.occultism.summon_horde_illager.conditions": "Not all requirements for this ritual are met.",
-  "ritual.occultism.summon_horde_illager.finished": "Summoned the small illager invasion successfully.",
-  "ritual.occultism.summon_horde_illager.interrupted": "Summoning of the small illager invasion interrupted.",
-  "ritual.occultism.summon_horde_illager.started": "Started summoning the small illager invasion.",
+  "ritual.occultism.summon_horde_illager.finished": "Summoned the wild illager invasion successfully.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2301]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 6]
```diff
@@ -2294,9 +2298,9 @@
...
+  "ritual.occultism.summon_horde_illager.interrupted": "Summoning of the wild illager invasion interrupted.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2302]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 7]
```diff
@@ -2294,9 +2298,9 @@
...
+  "ritual.occultism.summon_horde_illager.started": "Started summoning the wild illager invasion.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2303]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 8]
```diff
@@ -2294,9 +2298,9 @@
...
   "ritual.occultism.summon_marid_crusher.conditions": "Not all requirements for this ritual are met.",
   "ritual.occultism.summon_marid_crusher.finished": "Summoned marid ore crusher successfully.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2305]: Capitalization

File: src/generated/resources/assets/occultism/lang/en_us.json [hunk 9]
```diff
@@ -2294,9 +2298,9 @@
...
   "ritual.occultism.summon_marid_crusher.interrupted": "Summoning of marid ore crusher interrupted.",
```
Comment (MuteTiefling) [inline src/generated/resources/assets/occultism/lang/en_us.json:2306]: Capitalization

**Pull request merged**